doc/ss_logger.rst

```diff
@@ -0,0 +1,172 @@
+***************
+SmartSim Logger
+***************
+========
+Overview
+========
+SmartSim supports experiment tracking through logging functionality
+offered by the SmartSim `log` module. The logger, supported by Python logging, enables
+monitoring the experiment at runtime by allowing users to print messages **to stdout**
+and/or **to file** from within the Python driver script. The SmartSim logger permits users to categorize
+messages by severity level. Users may instruct SmartSim to print to stdout
+certain severity level log messages and omit others through the `SMARTSIM_LOG_LEVEL`
+environment variable. The `SMARTSIM_LOG_LEVEL` environment variable may be overridden when logging to file
+by specifying a log level to the ``log_to_file()`` function. Examples walking through
+both logging :ref:`to stdout<log_to_stdout>` and :ref:`to file<log_to_file>` are provided below.
+
+SmartSim offers **four** log functions to use within the Python driver script. The
+below functions accept string messages:
+
+- ``logger.warning()``
+- ``logger.error()``
+- ``logger.info()``
+- ``logger.debug()``
+
+The `SMARTSIM_LOG_LEVEL` environment variable accepts **three** log levels: `quiet`,
```

Comment:
Doesn't it also accept `developer`? Even if we don't use it, I think the level is still there